utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -35,22 +32,22 @@ object LicenseFilenamePatterns {
         "*.license",
         "unlicence",
         "unlicense"
-    ).generateCapitalizationVariants()
+    )
```

Comment:
I'm not sure I like this change. IMO any users of these lists, including future uses of the lists outside of VCS classes, should be able to rely on the list to contain license file names, for both case-sensitive and case-insensitive file systems.

Reply:
We use the matching currently only for matching license and patent files and I believe for this use case
case-insensitive matching is a better fit. In fact we already do case-insensitve matching in some places.

Anyhow, I'm doing this for preparing making license file patterns configurable.
An it made things a bit more complicated, this is why I would like to git rid of the case insensitive matching for now.
As soon as the configurability has been implemented it would be easy to add it again.

So would you be ok with doing this change now - and if needed by "future uses of the lists outside of VCS classes" we can add it in again?